index.js

```diff
@@ -113,4 +117,76 @@ app.get('/squadStats', async (req, res) => {
 
     res.send(retVal);
 
-});
\ No newline at end of file
+});
+
+//stats squad individual
+app.get('/squadStatsIndiv', async (req, res) => {
+
+    let gt = req.query.gt;
+
+    let result = await lib.halo.mcc['@0.0.11'].squad.activity({
+        gamertags: [
+            'sagwasswagga',
+            'Cragle 3',
+            'A Pregnant Nunn',
+            'babaganoosh069',
+            'Girthy Jello',
+            'SirPuffingtin',
+            'ya boi nar',
+            'l swordzzz'
+        ],
+        minSquadSize: '3',
+        game: 'Halo 3'
+    });
+
+
+    let games = result.games;
```

Comment:
how many games are in this result? can we specify amount

Reply:
unfortunately cannot specify, because it will return any games that match the criteria, so this number will grow when we play more games. but we can get the length from result.games.length